src/kmrbase.c

```diff
@@ -24,6 +24,13 @@
 #include "kmr.h"
 #include "kmrimpl.h"
 
+#define KMR_TRACE_ENABLE 1
```

Comment:
Do not use macro, use option of struct kmr.
Implement an option, "trace_enable", to the kmr struct so that the tracer is enabled only when kmr.trace_enabled is set 1.  Refer to some options already implemented, like 'ckpt_enable' and 'log_traces', etc.